packages/byods-demo-server/src/server.ts

```diff
@@ -0,0 +1,183 @@
+import express, {Request, Response, NextFunction} from 'express';
+import logger from 'jet-logger';
+import * as BYODS from '@webex/byods/src/index';
+import 'express-async-errors';
+import path from 'path';
+import helmet from 'helmet';
+import morgan from 'morgan';
+import cookieParser from 'cookie-parser';
+import handlebars from 'handlebars';
+import fs from 'fs';
+
+// **** Core Setup **** //
+const app = express();
+
+let sdk = new BYODS.BYODS({
+  clientId: 'your-client-id',
+  clientSecret: 'your-client-secret',
+  tokenStorageAdapter: undefined,
+  logger: {level: BYODS.LOGGER.INFO},
+});
+let baseClient = sdk.getClientForOrg('temp');
+
+// **** Helper Functions **** //
+// Function to render Handlebars templates
+function renderTemplate(res: Response, templateName: string, data: any) {
+  const templatePath = path.join(__dirname, 'views', `${templateName}.hbs`);
+  fs.readFile(templatePath, 'utf-8', (err, source) => {
+    if (err) {
+      res.status(500).send('Error reading template');
+
+      return;
+    }
+    const template = handlebars.compile(source);
+    const html = template(data);
+    res.send(html);
+  });
+}
+
+// **** Setup **** //
+
+// Basic middleware
+app.use(express.json());
+app.use(express.urlencoded({extended: true}));
+app.use(cookieParser('your-cookie-secret'));
+
+// Show routes called in console during development
+if (process.env.NODE_ENV === 'development') {
```

Comment:
These are all code that came with the `express-generator` output. Can remove it but thought it might be useful in the future.